sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -204,6 +259,48 @@ static const struct snd_soc_acpi_adr_device cs35l56_3_lr_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device cs35l56_1_3amp_adr[] = {
+	{
+		.adr = 0x00013001fa355601ull,
+		.num_endpoints = 1,
+		.endpoints = &spk_1_endpoint,
+		.name_prefix = "AMP1"
+	},
+	{
+		.adr = 0x00013101fa355601ull,
+		.num_endpoints = 1,
+		.endpoints = &spk_2_endpoint,
+		.name_prefix = "AMP2"
+	},
+	{
+		.adr = 0x00013201fa355601ull,
+		.num_endpoints = 1,
+		.endpoints = &spk_3_endpoint,
+		.name_prefix = "AMP3"
+	}
+};
+
+static const struct snd_soc_acpi_adr_device cs35l56_3_3amp_adr[] = {
```

Comment:
Thinking about that card I don't think we have a concept of a left and a right so it's just the "3amp" as the unique identifier which is fine by me

Reply:
Should we call this assumption out as an inline comment.

Reply:
@lgirdwood There is no assumption here. We introduce the amp's adr and their positions here. The naming here is to make sure we will have unique name for each acpi adr device array.